docs/event.md

```diff
@@ -43,6 +43,29 @@ is fired. The following key values are also supported:
  - `%arguments` - The arguments passed when the event was dispatched/triggered.
 
 
+@signature `(SCOPE_KEY_LOOKUP SCOPE_EVENT)='CALL_EXPRESSION'`
+
+Listens to an event in the [can-stache/expressions/key-lookup specified scope] and calls the [can-stache/expressions/call] when that event occurs.
+
+```
+<my-component (. someevent)="doSomething()"/>
```

Comment:
can we use something other than `.`?  Maybe show a bit more of an example to make the context more clear?